osquery/management.py

```diff
@@ -0,0 +1,144 @@
+"""LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import socket
+
+try:
+    import argparse
+except ImportError:
+    print("Cannot import argparse: pip install argparse")
+    exit(1)
+
+try:
+    from thrift.transport import TSocket
+    from thrift.transport import TTransport
+    from thrift.protocol import TBinaryProtocol
+    from thrift.server import TServer
+except ImportError:
```

Comment:
argparse and thrift are installed via setup.py, so this should never be hit. also, not sure if an exit is more useful than an ImportError